### PR TITLE
Zeroptian Invasion

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Zeroptian Invasion

## Game Information
- **Title**: Zeroptian Invasion
- **URL**: https://josyan.itch.io/zeroptian-invasion

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/zeroptianinvasion/port.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "genres": [
+      "arcade"
+    ],
+    "image": null,
+    "rtr": false,
+    "exp": false,
+    "runtime": null,
+    "reqs": [],
+    "arch": [
+      "armhf"
```
Comment (Cebion) [inline ports/zeroptianinvasion/port.json:28]: Is this a 32 bit port? If yes it also needs 32 bit export in the .sh
else this is wrong.

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/zeroptianinvasion/port.json
Comment (Fraxinus88) [inline ports/zeroptianinvasion/port.json:1]: arch should be aarch64 for gmloader next?

Comment (Ganimoth) [reply ports/zeroptianinvasion/port.json:1]: its 32bit, but it did not work for him on Arkos with the old gmloader.

File: ports/zeroptianinvasion/port.json (hunk 1 above)
Comment (cdeletre) [reply ports/zeroptianinvasion/port.json:28]: This one is aarch64

```
file gmloadernext
gmloadernext: ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, BuildID[sha1]=7a9e37b2cb8bdbda179b3d48b1c4ed65ba0ebc44, for GNU/Linux 3.7.0, not stripped
```

Comment (cdeletre) [inline ports/zeroptianinvasion/port.json:28]:
```suggestion
      "aarch64"
```

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/zeroptianinvasion/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<gameList>
+  <game>
+    <path>./Zeroptian Invasion.sh</path>
+    <name>Zeroptian Invasion</name>
+    <desc>Zeroptian Invasion is a classic shoot 'em up with mechanics from other arcades of the 80s. Travel back in time with this unique pixelated experience featuring an incredible soundtrack in chiptune.</desc>
+    <releasedate>20181128T000000</releasedate>
+    <developer>Joysan</developer>
+    <publisher>Self-Published</publisher>
+    <genre>Arcade</genre>
+    <image>./zeroptianinvasion/screenshot.png</image>
```
Comment (cdeletre) [inline ports/zeroptianinvasion/gameinfo.xml:11]: zeroptianinvasion-fs8
What do you think ? :)

File: ports/zeroptianinvasion/port.json (hunk 1 above)
Comment (tabreturn) [reply ports/zeroptianinvasion/port.json]: Oops! First gmloader-next port for me. Corrected

File: ports/zeroptianinvasion/gameinfo.xml (hunk 2 above)
Comment (tabreturn) [reply ports/zeroptianinvasion/gameinfo.xml]: Thanks. Looks good. Added as cover.png

**Review by Cebion — APPROVED** [review]

File: ports/zeroptianinvasion/zeroptianinvasion/game.apk
Comment (JanTrueno) [inline ports/zeroptianinvasion/zeroptianinvasion/game.apk:1]: Apk is abnormally sized. This might still has game assets in it.

**Review by JanTrueno — CHANGES_REQUESTED** [review]

File: ports/zeroptianinvasion/zeroptianinvasion/game.apk
Comment (JanTrueno) [reply ports/zeroptianinvasion/zeroptianinvasion/game.apk:1]: Thanks for checking, then its fine. Could potentially remove the 32bit libs to save some space.

Comment (Fraxinus88) [reply ports/zeroptianinvasion/zeroptianinvasion/game.apk:1]: Does the game.apk need to be zipped with -0 compression?

Comment (JanTrueno) [reply ports/zeroptianinvasion/zeroptianinvasion/game.apk:1]: For the gamemaker libraries it doesnt seem to matter. Only audio seems to be a problem. So it depends if you repack the OGG's.

**Review by JanTrueno — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**